### PR TITLE
[Dependency] Upgrade Ignite to 2.9.0

### PR DESCRIPTION
**Pull request by songxiaoyan**
Update iginte to 2.9.0

### CODE REVIEW

File: legacy/pom.xml [hunk 1]
```diff
@@ -124,7 +124,7 @@
         <dependency>
             <groupId>org.apache.ignite</groupId>
             <artifactId>ignite-core</artifactId>
-            <version>2.8.1</version>
+            <version>2.9.0</version>
```
Comment (songxiaoyan) [inline legacy/pom.xml:127]: test comment

Comment (xieus) [reply legacy/pom.xml:127]: it works :-)

**Comment by xieus** [comment]
@songxiaoyan Congratulations to your first PR. Short but timely.

**Comment by xieus** [comment]
You can find the CI failure on this page https://github.com/futurewei-cloud/alcor/pull/503/checks?check_run_id=1491312470. Alternatively, you can click "Details" link on the row of failed Java CI. 

You should be able to reproduce the same error on your local machine (following the instructions on https://github.com/futurewei-cloud/alcor/pull/503/checks?check_run_id=1491312470).  It is related to Security Group manager.

**Comment by xieus** [comment]
@chenpiaoping Please review Xiaoyan's PR as well. Thanks.

**Comment by songxiaoyan** [comment]
> @chenpiaoping Please review Xiaoyan's PR as well. Thanks.



> You can find the CI failure on this page https://github.com/futurewei-cloud/alcor/pull/503/checks?check_run_id=1491312470. Alternatively, you can click "Details" link on the row of failed Java CI.
> 
> You should be able to reproduce the same error on your local machine (following the instructions on https://github.com/futurewei-cloud/alcor/pull/503/checks?check_run_id=1491312470). It is related to Security Group manager.

**Comment by songxiaoyan** [comment]
> You can find the CI failure on this page https://github.com/futurewei-cloud/alcor/pull/503/checks?check_run_id=1491312470. Alternatively, you can click "Details" link on the row of failed Java CI.
> 
> You should be able to reproduce the same error on your local machine (following the instructions on https://github.com/futurewei-cloud/alcor/pull/503/checks?check_run_id=1491312470). It is related to Security Group manager.

Okey,I'm looking at this mistake.But I passed the local test before.

**Review by chenpiaoping — APPROVED** [review]

**Pull request merged**